usaspending_api/common/matview_manager.py

```diff
@@ -11,7 +11,10 @@
 DEPENDENCY_FILEPATH = APP_DIR / "database_scripts/matviews/functions_and_enums.sql"
 JSON_DIR = APP_DIR / "database_scripts/matview_sql_generator"
 MATVIEW_GENERATOR_FILE = APP_DIR / "database_scripts/matview_generator/matview_sql_generator.py"
-OVERLAY_VIEWS = [APP_DIR / "database_scripts/matviews/vw_award_search.sql"]
+OVERLAY_VIEWS = [
+    APP_DIR / "database_scripts/matviews/vw_award_search.sql",
+    APP_DIR / "database_scripts/etl/transaction_delta_view.sql",
```

Comment:
This view is for the Elasticsearch ETL script.

Was it necessary for any of the tests below?

Reply:
Was confused by this one. It wasn't needed for anything I touched, but I had a bunch of test cases fail on me when I caught up with dev on my branch. All failed because this view didn't exist.

Reply:
Odd

Reply:
Agreed. Here is the Travis log before I made the change from this comment: https://travis-ci.com/fedspendingtransparency/usaspending-api/jobs/241100378